castai/resource_node_template_test.go

```diff
@@ -414,11 +414,11 @@ func TestAccResourceNodeTemplate_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "custom_instances_enabled", "false"),
 					resource.TestCheckResourceAttr(resourceName, "custom_label.#", "0"),
 					resource.TestCheckResourceAttr(resourceName, "custom_labels.%", "2"),
-					resource.TestCheckResourceAttr(resourceName, "custom_labels.custom-key-1", "custom-value-1"),
-					resource.TestCheckResourceAttr(resourceName, "custom_labels.custom-key-2", "custom-value-2"),
+					resource.TestCheckResourceAttr(resourceName, fmt.Sprintf("custom_labels.%s-1", rName), fmt.Sprintf("%s-1", rName)),
+					resource.TestCheckResourceAttr(resourceName, fmt.Sprintf("custom_labels.%s-2", rName), fmt.Sprintf("%s-2", rName)),
 					resource.TestCheckResourceAttr(resourceName, "custom_taints.#", "1"),
-					resource.TestCheckResourceAttr(resourceName, "custom_taints.0.key", "custom-taint-key-1"),
-					resource.TestCheckResourceAttr(resourceName, "custom_taints.0.value", "custom-taint-value-1"),
+					resource.TestCheckResourceAttr(resourceName, "custom_taints.0.key", fmt.Sprintf("%s-1", rName)),
+					resource.TestCheckResourceAttr(resourceName, "custom_taints.0.value", fmt.Sprintf("%s-1", rName)),
```

Comment:
taint key and value was different values before

Reply:
Fixed